### PR TITLE
[3750] [1617] Error message for not long enough password

### PR DESCRIPTION
**Pull request by Dagnija-Viktorova**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3750
* Fixes https://github.com/scandipwa/scandipwa/issues/1617

**Problem:**
* No custom error message when entered password is less than 8 characters

**In this PR:**
* Added to validation rules error message 'Password should be at least 8 characters long' with min range 8 to sign in, confirm account, reset password and change password forms

### CODE REVIEW

**Comment by Elena-Evsa** [comment]
Issue 1: field 'New password' in My Account > Account Information > Change Password has no same validation, 
Field 'New password' stay green and error message appear
Field should become red and error message appear under the field
https://vimeo.com/656995356/89f8a2b7c8
Issue 2: validation absent on confirm account page 
image

**Comment by Elena-Evsa** [comment]
Issue 1: Passed
Issue 2: please add word 'characters' after number of symbols everythere after 'Password should be at least Х' 
image

**Comment by Dagnija-Viktorova** [comment]
Changed password length fail message as requested, can it pass QA?

**Comment by Elena-Evsa** [comment]
Please, take a look on Issue 3

File: packages/scandipwa/src/component/PasswordChangeForm/PasswordChangeForm.form.js [hunk 1]
```diff
@@ -30,7 +30,14 @@ export const customerEmailAndPasswordFields = () => [
         validateOn: ['onChange'],
         validationRule: {
             isRequired: true,
-            inputType: VALIDATION_INPUT_TYPE.password
+            inputType: VALIDATION_INPUT_TYPE.password,
+            showErrorAsLabel: true,
+            customErrorMessages: {
+                onRangeFailMin: `Password should be at least ${props.minimumPasswordLength} characters`
```
Comment (tatiana-scandi) [inline packages/scandipwa/src/component/PasswordChangeForm/PasswordChangeForm.form.js:36]: Why not just pass `minimumPasswordLength` as function parameter?

File: packages/scandipwa/src/route/ConfirmAccountPage/ConfirmAccountPage.component.js [hunk 2]
```diff
@@ -91,7 +93,14 @@ export class ConfirmAccountPage extends PureComponent {
                   validateOn={ ['onChange'] }
                   validationRule={ {
                       isRequired: true,
-                      inputType: VALIDATION_INPUT_TYPE.password
+                      inputType: VALIDATION_INPUT_TYPE.password,
+                      showErrorAsLabel: true,
+                      customErrorMessages: {
+                          onRangeFailMin: `Password should be at least ${minimumPasswordLength} characters`
```
Comment (tatiana-scandi) [inline packages/scandipwa/src/route/ConfirmAccountPage/ConfirmAccountPage.component.js:99]: Please, use a translatable string here.

File: packages/scandipwa/src/component/PasswordChangeForm/PasswordChangeForm.form.js (hunk 1 above)
Comment (tatiana-scandi) [reply packages/scandipwa/src/component/PasswordChangeForm/PasswordChangeForm.form.js:36]: + Please, use a translatable string here.

File: packages/scandipwa/src/component/MyAccountSignIn/MyAccountSignIn.component.js [hunk 3]
```diff
@@ -76,7 +78,14 @@ export class MyAccountSignIn extends PureComponent {
                   validateOn={ ['onChange'] }
                   validationRule={ {
                       isRequired: true,
-                      inputType: VALIDATION_INPUT_TYPE.password
+                      inputType: VALIDATION_INPUT_TYPE.password,
+                      showErrorAsLabel: true,
+                      customErrorMessages: {
+                          onRangeFailMin: `Password should be at least ${minimumPasswordLength} characters`
```
Comment (tatiana-scandi) [inline packages/scandipwa/src/component/MyAccountSignIn/MyAccountSignIn.component.js:84]: Please, use a translatable string here + think of a way to reuse password fields + validation logic (now there is too much duplicated code)

**Review by tatiana-scandi — CHANGES_REQUESTED** [review]
CR failed.

**Comment by carinadues** [comment]
Issue will be fixed in https://github.com/scandipwa/scandipwa/pull/4145

**Pull request closed without merging**